scalabel/tests/test_scalabel_api.py

```diff
@@ -0,0 +1,73 @@
+""" Test file for the scalabel api """
+import subprocess
+import time
+import logging
+import os
+import signal
+import shutil
+import pytest
+import yaml
+from urllib3.exceptions import HTTPError
+from ..scalabel_api import ScalabelAPI
```

Comment:
Use full path,
"from scalabel.scalabel_api import ScalabelAPI"

Add the scalabel to the appropriate python path:
export PYTHONPATH="..."

Reply:
Changed to full path, but didn't need to modify python path for it to work